Tests/FluentAssertions.Specs/Xml/XDocumentAssertionSpecs.cs

```diff
@@ -1111,5 +1111,105 @@ public void When_asserting_a_document_has_a_null_element_it_should_fail()
         }
 
         #endregion
+
+        #region HaveSingleElement
+
+        [Fact]
+        public void When_asserting_document_has_a_single_child_element_and_it_does_it_should_succeed()
```

Comment:
🔧 I suggest to keep the name more functional and concise: `A_single_expected_child_element_with_the_specified_name_should_be_accepted`